homeassistant/components/miele/strings.json

```diff
@@ -877,6 +877,28 @@
       "superfreezing": {
         "name": "Superfreezing"
       }
+    },
+    "vacuum": {
+      "vacuum": {
+        "name": "Vacuum",
```

Comment:
Why isn't the name None? I would assume that this entity would be the main feature of the devuce